ledger-tool/src/output.rs

```diff
@@ -594,21 +599,17 @@ impl AccountsOutputStreamer {
 
     pub fn output(&self) {
         match self.output_format {
-            OutputFormat::Json | OutputFormat::JsonCompact => self.output_json(),
+            OutputFormat::Json | OutputFormat::JsonCompact => self.output_json().unwrap(),
             _ => self.print(),
         }
     }
 
-    fn output_json(&self) {
+    fn output_json(&self) -> Result<(), serde_json::Error> {
```

Comment:
prefer avoiding dependency/external types in our APIs

Reply:
Fair enough; ended up mapping these these to a `String` in https://github.com/solana-labs/solana/pull/34915/commits/a655c45a74619817069a9c21d495a80e9f4104a5. Hypothetically, I could make a subfunction for json, use `?` within it and then do a single `.unwrap()` or `.map_err()` on it. There are only 4 `.map_err()`'s so doesn't seem awful as I just pushed it in